chef_master/source/config_rb_server_optional_settings.rst

```diff
@@ -178,7 +185,9 @@ data_collector
 This configuration file has the following settings for ``data_collector``:
 
 ``data_collector['root_url']``
-   The fully qualified URL to the data collector server API. When present, it will enable the data collector **opscode-erchef** application.
+   The fully qualified URL to the data collector server API. When present, it will enable the data collector in **opscode-erchef**. This also enables chef-server authenticated forwarding Any properly signed requests arriving at /organizations/ORGNAME/data-collector will be forwarded to this URL. This is also target for requests authenticated and forwarded by the ``/organizations/ORGNAME/data-collector`` endpoint.
```

Comment:
The only minor thing I would add is that you need to configure `root_url` and `token` for this to happen.